topbeat/beat/sigar.go

```diff
@@ -15,113 +15,41 @@ type SystemLoad struct {
 }
 
 type CpuTimes struct {
-	User          uint64  `json:"user"`
+	Cpu           sigar.Cpu
```

Comment:
I thought that might be the reason. So if you embed `sigar.Cpu` into `CpuTimes` you can set all the fields at once. See this playground example I put together http://play.golang.org/p/14aEfQiS30